src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    public static final String LINE = "____________________________________________________________";
+
+    public static void printList(Task[] list) {
+        System.out.println(LINE);
+        if (list[0] == null) {
+            System.out.println("No items added!");
+        }
+        int i = 0;
+        while (list[i] != null) {
+            System.out.println((i+1) + ". " + list[i]);
+            i += 1;
+        }
+        System.out.println(LINE);
+    }
```

Comment:
I like how you separated each outputs into different methods. It makes it easier for me to read the code.